api/fixtures/example.go

```diff
@@ -54,7 +55,7 @@ type ExampleOptions struct {
 	NodePoolReplicas                 int32
 	InfraID                          string
 	ComputeCIDR                      string
```

Comment:
can we align these terms ServiceCIDR, ComputeCIDR and PodCIDR with the ones you are introducing, so it'd be MachineCIDR and ClusterCIDR? or the other way around I don't mind but let's use the same everywhere.

Reply:
sure, good point.